addons/bus/static/src/js/services/async_service.js

```diff
@@ -0,0 +1,41 @@
+odoo.define('bus.asyncService', function (require) {
+"use strict";
+
+const AbstractService = require('web.AbstractService');
+const { serviceRegistry, bus, _lt, _t } = require('web.core');
+
+
+const notifHead = _lt("Background task created");
+const notifCreatedBody = _lt("The task %s has been scheduled for processing and will start shortly.");
+
+const asyncService = AbstractService.extend({
+    dependencies: ['bus_service', 'ajax', 'crash_manager'],
+    start() {
+        this._super(...arguments);
+        this.call('bus_service', 'onNotification', this, this._onNotification);
+        bus.on('async_job_created', this, (jobId, jobName) => {
+            this.do_notify(notifHead, _.str.sprintf(""+notifCreatedBody, _t(jobName)));
+        });
+    },
+    rpc(jobName, params, options) {
+        params = params || {};
+        params.params = params.params || {};
+        params.params.context = params.params.context || {};
+        params.params.context.async_notify = jobName;
+
+        return this._rpc(params, options);
+    },
+    _onNotification(notifications) {
+        notifications.filter(notif => {
+            return notif[1]['type'] === 'ir.async';
+        }).forEach(notif => {
+            let {jobId, jobName, jobState, payload} = notif[1];
+            bus.trigger('async_job_' + jobState, jobId, jobName, payload && JSON.parse(payload));
```

Comment:
Don't really like dynamically build event names.
Prefer always triggering `async_job` and pass status as data.
This is especially relevant when `AsyncJob(Systray)Menu` always react to this as an `_updateJob` operation.